obspy/io/nlloc/core.py

```diff
@@ -321,6 +321,18 @@ def _read_single_hypocenter(lines, coordinate_converter, original_picks,
     ou.azimuth_max_horizontal_uncertainty = hor_unc_azim
     ou.preferred_description = str("uncertainty ellipse")
     ou.confidence_level = 68  # NonLinLoc in general uses 1-sigma (68%) level
+    if "QML_ConfidenceEllipsoid" in lines.keys():
```

Comment:
just a nit pick.. `if "QML_ConfidenceEllipsoid" in lines:` is shorter and more readable, but thats just cosmetics, really